lib/presentation/pages/presenter.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:healthy_routine_mobile/healthy_routine.dart';
+import 'package:flutter/material.dart';
+
+class Presenter {
+  static String taskName;
+  static Category category;
+  static List<Weekday> listWeekDay = List();
+  static TimeOfDay startTime;
+  static TimeOfDay endTime;
+  static bool repeat;
+
+  static void selectListWeekDay(Weekday day, bool selected) {
+    selected
+        ? listWeekDay.add(day)
+        : listWeekDay.remove(day);
+  }
+
+  static void saveTask() {
+    Task t = Task(
+        name: taskName,
+        status: TaskStatus.pending,
+        recurrence: listWeekDay,
+        endDate: DateTime.now(),
+        startDate: DateTime.now(),
```

Comment:
Shouldn't we use the values selected by the user?

Reply:
Yep, but the select dialog is not a DateTime is a TimeOfDay... I'll chage the model to be TimeOfDay